database/dbase_test.go

```diff
@@ -139,14 +139,16 @@ func TestReconnectableCloudtrustDB(t *testing.T) {
 	var mockDBFactory = mock.NewCloudtrustDBFactory(mockCtrl)
 	var expectedError = errors.New("error")
 
+	var mockLogger = mock.NewLogger(mockCtrl)
```

Comment:
Alternative that does not require to add go generate : var mockLogger = log.NewNopLogger()

Reply:
fixed